docs/testnet/example-dapps.md

```diff
@@ -0,0 +1,52 @@
+# Example Dapps on Testnet
+Obscuro Testnet includes an example dapp to help you better understand how dapps capitalise on Obscuro's unique privacy features.
+
+## Number Guessing Game
+[A number guessing game](http://testnet-obscuroscan.uksouth.azurecontainer.io/game.html) is a simple way of showcasing the principles of privacy in Obscuro. The goal of the game is to guess a secret number. Each time an attempt is made an entrance fee of 1 token is paid. If a player correctly guesses the number the contract will pay out all of the accumulated entrance fees to them and reset itself with a new random number.
+
+Consider for a moment how a secret number could be created without divulging it. This is impossible in a transparent, decentralised ecosystem like Ethereum because when the secret number is generated by the smart contract it would be trivial to look up the internal state of the contract and see the secret number. OK, so let's imagine the internal state of the contract and therefore the secret number _is_ hidden, what happens when players start making their guesses? Using a block explorer to see the attempted guesses in plain view within the details of the transaction immediately provides players with an unfair advantage and the game is ruined.
+
+Building the guessing game in Obscuro addresses both scenarios described above. The guessing game smart contract generates a random secret number when it is deployed. This number is never revealed to a player or node operator, or indeed anybody because it is generated within a Trusted Execution Environment. And when players make their guesses the transactions carrying the guesses are encrypted and therefore obscured in a block explorer.
+
+### How to Play
+1. Start up the wallet extension. Follow instructions [here](https://docs.obscu.ro/wallet-extension/wallet-extension.html) and start the wallet extension using the following flags:
+`--nodeHost=testnet.obscu.ro --nodePortHTTP=13000 --nodePortWS=13001`
```

Comment:
Include `--port 3000` in the list of command args?

Reply:
Strictly speaking this is not required because this is the default setting but I'll add for completeness